sig/lrama/grammar/parameterizing_rules/builder/list.rbs

```diff
@@ -0,0 +1,11 @@
+module Lrama
+  class Grammar
+    class ParameterizingRules
+      class Builder
+        class List < Base
+          def build: -> Array[Rule]
```

Comment:
I think these sub classes inherit type definition of `#build` from `Base` class. Is this, and following type definition of `#build`, needed?

Reply:
Indeed it is. Thank you! I deleted it.